api/src/main/java/org/openmrs/module/appointmentscheduling/AppointmentDailyCount.java

```diff
@@ -0,0 +1,60 @@
+package org.openmrs.module.appointmentscheduling;
+
+import org.openmrs.BaseOpenmrsData;
+
+import java.io.Serializable;
+import java.util.Date;
+import java.util.List;
+
+public class AppointmentDailyCount extends BaseOpenmrsData implements Serializable {
```

Comment:
@jecihjoy - something that implements OpenmrsData is typically data that is saved to the database, and which has an id, uuid, etc.  This is more of a convenience bean.  I don't think it should be an OpenmrsObject.